src/protocolgame.cpp

```diff
@@ -1895,23 +1895,22 @@ void ProtocolGame::sendStoreBalance()
 	writeToOutputBuffer(msg);
 }
 
-void ProtocolGame::sendMarketEnter(uint32_t depotId)
+void ProtocolGame::sendMarketEnter()
 {
 	NetworkMessage msg;
 	msg.addByte(0xF6);
 	msg.addByte(std::min<uint32_t>(IOMarket::getPlayerOfferCount(player->getGUID()), std::numeric_limits<uint8_t>::max()));
 
-	DepotChest* depotChest = player->getDepotChest(depotId, false);
-	if (!depotChest) {
-		msg.add<uint16_t>(0x00);
-		writeToOutputBuffer(msg);
-		return;
-	}
-
 	player->setInMarket(true);
 
 	std::map<uint16_t, uint32_t> depotItems;
-	std::forward_list<Container*> containerList { depotChest, player->getInbox() };
+	std::forward_list<Container*> containerList { player->getInbox() };
```

Comment:
```suggestion
	std::forward_list<Container*> containerList{player->getInbox()};
```

Make a vector if possible too.